mempool/cat/store_test.go

```diff
@@ -40,85 +40,6 @@ func TestStoreSimple(t *testing.T) {
 	require.Nil(t, store.get(key))
 	require.Zero(t, store.size())
 	require.Zero(t, store.totalBytes())
-	require.Empty(t, store.orderedTxs)
-	require.Empty(t, store.txs)
-}
-
-func TestStoreOrdering(t *testing.T) {
```

Comment:
are these planned to be updated or just deleted?

Reply:
part of the revert, they were only added to test reaping txs out of order